src/App.jsx

```diff
@@ -17,7 +17,7 @@ function App() {
         <Route path= "/" element= {<Home />}/>
         <Route path= "/articles" element= {<Articles />} />
         <Route path= "/topics" element= {<Topics />} />
-        <Route path= "/articlesByTopic/:TopicName" element= {<ArticlesByTopic />} />
+        <Route path= "/articlesInTopic/:topic" element= {<ArticlesByTopic />} />
```

Comment:
not a massive fan of this route name i would prefer '/articles/topic/:topic' or 'topics/:topic' to be a bit more streamlined, we never see a URL that has XinY kind of naming convention